Fix for titles not updating

DavidVadnais opened the pull request:
"fix" for issue #1168. This works but I doubt this is the intended solution.  Let me know how you want to handle ticket_title vs title going forward and I can make the needed changes.

This is the current staff.py call for this (below). However update tickets has options for both `title` and `ticket_title`.
```python
    update_ticket(
        request.user,
        ticket,
        title = title,
        comment = comment,
        files = request.FILES.getlist('attachment'),
        public = request.POST.get('public', False),
        owner = int(request.POST.get('owner', -1)),
        priority = int(request.POST.get('priority', -1)),
        queue = int(request.POST.get('queue', -1)),
        new_status = new_status,
        time_spent = get_time_spent_from_request(request),
        due_date = get_due_date_from_request_or_ticket(request, ticket),
        new_checklists = new_checklists,
        customfields_form = customfields_form,
    )
```

edit: for linking Fixes #1168

Benbb96 commented:
From what I'm seeing, I think you can remove `ticket_title` from parameters because it does not seem to be used.

DavidVadnais commented:
Summary:
Replaced the `ticket_title` field with `title`. Updated all relevant references found.

Testing:
Validated functionality using my local test rig, built as an extension of another Django project. Confirmed the changes work as expected.

Environment:
    OS: Ubuntu 22.04
    Browser: Firefox
